lib/web_ui/lib/src/engine/html/render_vertices.dart

```diff
@@ -613,79 +744,100 @@ class _GlContext {
       final int bufferWidth = _widthInPixels!;
       final int bufferHeight = _heightInPixels!;
```

Comment:
The three lines above can be moved out of the if/else block and shared between the branches.

Reply:
Done